helm-charts/chatqna/gaudi-values.yaml

```diff
@@ -31,16 +31,17 @@ securityContext:
 
 # To override values in subchart llm-uservice
 llm-uservice:
+  TGI_LLM_ENDPOINT: ""
```

Comment:
maybe we should not set any thing for llm-uservice here, if the default one works for us